known.json

```diff
@@ -19,5 +19,6 @@
   	"15441526754256992144L" : { "owner" : "HitBTC.com", "description" : "Hot Wallet" },
 	"2797034409072178585L" 	: { "owner" : "BitBay.net", "description" : "Withdrawal Wallet" },
 	"15610359283786884938L" : { "owner" : "Binance", "description" : "Hot Wallet" },
-	"14367838290034827614L" : { "owner" : "Coincheck.com", "description" : "Main Wallet" }
+	"14367838290034827614L" : { "owner" : "Coincheck.com", "description" : "Main Wallet" },
+	"16313739661670634666L" : { "owner" : "Explorer Account", "description" : "Known addresses test"}
```

Comment:
I think we shouldn't mix Testnet addresses with Mainnet in one file.
Another solution would be to create a separate file for testnet wallets.

Reply:
Yeah, I've just noticed this change broke e2e test.
I'll think a better way to test that.